README.md

```diff
@@ -12,6 +12,17 @@ If something doesn't work, then feel free to [submit an issue](https://github.co
 
 More reading about protocol handlers: http://pla.nette.org/en/how-open-files-in-ide-from-debugger.
 
+Installing on Linux
+===================
+
+> If you use Arch Linux (Arch, Manjaro, Antergos, etc.), you can install [phpstorm-url-handler](https://aur.archlinux.org/packages/phpstorm-url-handler/) package.
+
+1. clone repository [https://github.com/sanduhrs/phpstorm-url-handler](https://github.com/sanduhrs/phpstorm-url-handler)
```

Comment:
These instructions are basically copy/paste from mentioned repo. It would be better not to copy/paste because instructions on external repo might change over time and then this repo will only display outdated instructions.